docs/source/cpp/compute.rst

```diff
@@ -255,41 +255,52 @@ Input(s) will be cast to the :ref:`common numeric type <common-numeric-type>`
++--------------------------+------------+--------------------+---------------------+-------+
+| add                      | Binary     | Numeric            | Numeric             |       |
++--------------------------+------------+--------------------+---------------------+-------+
+| add_checked              | Binary     | Numeric            | Numeric             |       |
++--------------------------+------------+--------------------+---------------------+-------+
+| divide                   | Binary     | Numeric            | Numeric             |       |
++--------------------------+------------+--------------------+---------------------+-------+
+| divide_checked           | Binary     | Numeric            | Numeric             |       |
++--------------------------+------------+--------------------+---------------------+-------+
+| multiply                 | Binary     | Numeric            | Numeric             |       |
++--------------------------+------------+--------------------+---------------------+-------+
+| multiply_checked         | Binary     | Numeric            | Numeric             |       |
++--------------------------+------------+--------------------+---------------------+-------+
+| negate                   | Unary      | Numeric            | Numeric             |       |
++--------------------------+------------+--------------------+---------------------+-------+
+| negate_checked           | Unary      | Signed Numeric     | Signed Numeric      |       |
++--------------------------+------------+--------------------+---------------------+-------+
+| power                    | Binary     | Numeric            | Numeric             |       |
++--------------------------+------------+--------------------+---------------------+-------+
+| power_checked            | Binary     | Numeric            | Numeric             |       |
++--------------------------+------------+--------------------+---------------------+-------+
+| sign                     | Unary      | Numeric            | Int8                | \(1)  |
++--------------------------+------------+--------------------+---------------------+-------+
+| sign_with_signed_zero    | Unary      | Numeric            | Int8                | \(2)  |
++--------------------------+------------+--------------------+---------------------+-------+
+| subtract                 | Binary     | Numeric            | Numeric             |       |
++--------------------------+------------+--------------------+---------------------+-------+
+| subtract_checked         | Binary     | Numeric            | Numeric             |       |
++--------------------------+------------+--------------------+---------------------+-------+
+
+* \(1) Output is an integral representing the signedness of the input.
+  For nonzero inputs, output is any of (-1,1) and 0 for zero input.
+
+* \(2) Similar to ``sign`` but zero input is considered as a signed number.
```

Comment:
```suggestion
* \(2) Similar to ``sign`` but zero is considered signed for floating point inputs.
```

Reply:
The `sign_with_signed_zero` variant treats 0 as a positive value for integral types, but either -0 or +0 for floating-point types.

Reply:
Hmm...I agree with this change. The `sign_with_signed_zero` variant only applies to floating-point numbers resulting in either (-1,1). For integral values, the result will be either of (-1,0,1).